components/styles/styler.component.ts

```diff
@@ -233,24 +241,33 @@ export class HsStylerComponent {
   setStyleByJson(style_json: StyleJson): void {
     const style = new Style(style_json);
     const layer = this.HsStylerService.layer;
-    const isClustered = this.HsLayerUtilsService.isLayerClustered(layer);
+    // const isClustered = this.HsLayerUtilsService.isLayerClustered(layer);
     switch (this.level) {
       case 'feature':
         this.setStyleForFeatures(layer, style);
         break;
+      case 'cluster':
+        this.HsLayerEditorVectorLayerService.clusterStyle.setFill(
+          style.getImage().getFill()
+        );
+        this.HsLayerEditorVectorLayerService.clusterStyle.setStroke(
+          style.getImage().getStroke()
+        );
+        this.HsLayerEditorVectorLayerService.styleLayer(layer);
+        break;
       case 'layer':
-      default:
         this.setStyleForFeatures(layer, null);
-        if (isClustered) {
+        if (this.isClustered) {
           /* hsOriginalStyle is used only for cluster layers 
           when styling clusters with just one feature in it */
           this.HsStylerService.layer.set('hsOriginalStyle', style);
         } else {
           layer.setStyle(style);
         }
         break;
+      default:
```

Comment:
I think having default option 'layer' was fine.

Reply:
Default applies when no other condition applies so when we have 'level: cluster | feature | layer = layer' , layer is still default. But surely I can change the code style

Reply:
5b52b0c